### ISSUE TITLE
Bugzilla attachments should live on GitHub

### ISSUE THREAD

**Issue by JasonGross** [coq/coq.github.io#234]
#### Description of the problem

Currently migrated bugs link to bugzilla attachments.  [GitHub supports uploading `.txt`, `.log`, and `.zip` files](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) as attachments on issues.  I propose that for all bugs with attachments on bugzilla, we archive the files in a `.zip` file, and attach that to the github issue.

One possible alternative is to special-case `.txt`, `.log`, and `.v` files, to rename `foo.v` to `foo.v.txt`, and upload these separately.  However, I don't think there's any particular benefit to doing this, as I don't think you can view `.txt` or `.log` file attachments in the browser without downloading them.

**Comment by Zimmi48** [comment]
With the GitHub Pages website migration, Bugzilla attachments are no longer available on the Coq website. It's time to decide whether to migrate them to GitHub attachments or to close this issue, based on the principle that, after many years, most of these issues have been solved and/or the attachments have become irrelevant, and that if they need to be retrieved, there is still the tarball provided by Pierre in this thread.

**Comment by JasonGross** [comment]
Does coqbot support uploading attachments to GitHub yet?  I think it would be nice but not essential to get the attachments on GitHub, I expect it shouldn't be too hard to script if we already have code in coqbot for uploading attachments and commenting with them, and furthermore we'll want the ability to upload attachments for enhancing the minimizer comments, so I don't think adding this functionality to coqbot would be wasted effort.  (It seems fine to go the route of making a new comment with "here's the attachment that was previously named ..." rather than editing existing comments, if that's easier to script.)

**Comment by Zimmi48** [comment]
No, GitHub still doesn't provide an official API for uploading attachment files. A [comment from 2021](https://github.com/isaacs/github/issues/1133#issuecomment-860554483) explained this. However, it seems that there exists an undocumented API that allows to do this. See: https://github.com/j178/github-s3. I'm not sure that abusing it in coqbot would be a good idea though.

**Comment by JasonGross** [comment]
> However, it seems that there exists an undocumented API that allows to do this. See: https://github.com/j178/github-sfile.  

Looking at the code, this is just some go code that pretends to be the browser doing the same thing that happens when you drag and drop a file

**Comment by Zimmi48** [comment]
Transferring the issue to the new website since the attachments from historical bugs on Bugzilla have not yet been migrated anywhere (besides the tarball shared by Pierre Letouzey in https://github.com/rocq-prover/coq.github.io/issues/234#issuecomment-1760235759).
